certificate-renewal/REPWRITERSPECS/BANNO.CDRENEW.V1.POW

```diff
@@ -1323,15 +1336,44 @@ PROCEDURE ERRORHANDLER
  NEWLINE
  PRINT INDENT(1)+Q+ERRORMSG+Q+": "+Q+ERRORDETAIL(ERRORDETAILOFFSET)+Q
 
- IF ERRORCODE=503 THEN
+ IF ERRORCODE=503 OR
+    ERRORCODE=505 THEN
   DO
    PRINT ","
    NEWLINE
-   CALL PRINTCURRENTSTATE
-   NEWLINE
-   PRINT INDENT(2)+ "}"
+   PRINT INDENT(1)+Q+"results"+Q+": {"
    NEWLINE
+
+   CALL PRINTCURRENTSTATE
+
+   IF ERRORCODE=505 THEN
+    DO
+[* Display ineligible cross account message
+*]
+     PRINT ","
+     NEWLINE
+     PRINT INDENT(2)+Q+"ineligibleCrossAccountMessage"+Q+": ["
```

Comment:
I see that `ineligibleCrossAccountMessage` is still output in `RETURNJSON1`.  It can probably be removed from that procedure.  The UX only needs that property when error code 505 is returned.  I guess it doesn't hurt to leave it in both places but it can be safely removed (from `RETURNJSON1`).

Reply:
removed